types/store/rhs.ts

```diff
@@ -24,6 +24,10 @@ export type PostDraft = {
     uploadsInProgress: string[];
     props?: any;
     caretPosition?: number;
+    channelId: string;
+    createAt: Date;
+    updateAt: Date;
```

Comment:
I don't think we can use Date objects here since they're written to storage/the indexed DB which I think requires encoding them as JSON which breaks the date. If that's the case, we could probably use numbers and store the unix timestamps like we do on the server

Reply:
Yep, at first I used whatever was used for the timestamp which was date object, but yep it should be a number, will change, thanks!